packages/amplication-server/src/core/github/github.service.ts

```diff
@@ -17,11 +15,24 @@ import { GitUser } from '../git/dto/objects/GitUser';
 import { GithubFile } from './dto/githubFile';
 import { GithubRepo } from './dto/githubRepo';
 import { GithubTokenExtractor } from './utils/tokenExtractor/githubTokenExtractor';
+import { CreateGitOrganizationArgs } from '../git/dto/args/CreateGitOrganizationArgs';
+import { PrismaService } from 'nestjs-prisma';
```

Comment:
github service (provider) should not be aware of Prisma or the DB.

Reply:
Done